rock-paper-scissors.py

```diff
@@ -20,12 +21,77 @@
 def mapper(value):
     return REV_CLASS_MAP[value]
 
+
+def winner(move_by_user, move_by_cmp):
```

Comment:
could have handled it better by using `if-else` blocks instead multiple `if` blocks. Makes the code very inefficient.